crates/editor/src/selections_collection.rs

```diff
@@ -426,46 +426,6 @@ impl<'a> MutableSelectionsCollection<'a> {
         self.selections_changed = true;
     }
 
-    pub(crate) fn set_pending_display_range(
-        &mut self,
-        range: Range<DisplayPoint>,
-        mode: SelectMode,
-    ) {
-        let (start, end, reversed) = {
-            let display_map = self.display_map();
-            let buffer = self.buffer();
-            let mut start = range.start;
-            let mut end = range.end;
-            let reversed = if start > end {
-                mem::swap(&mut start, &mut end);
-                true
-            } else {
-                false
-            };
-
-            let end_bias = if end > start { Bias::Left } else { Bias::Right };
```

Comment:
The culprit is here: `let end_bias = Bias::Right` since `end == start`, and this is the only usage of this method. So, I'm thinking of using the more direct `set_pending_anchor_range` and removing this method.